pom.xml

```diff
@@ -335,6 +336,11 @@
           <version>2.17</version>
           <dependencies>
             <dependency>
+              <groupId>org.apache.apex</groupId>
+              <artifactId>apex-codestyle-config</artifactId>
+              <version>${project.version}</version>
```

Comment:
I am not sure of this but can we not change this to a fixed previously released version?